website/docs/user-group-role/user/user_basic_operations.md

```diff
@@ -53,32 +50,38 @@ To view details about a specific user:
 
 After the creation of the user, you can edit any parameter defined during the creation.
 
-To modify a user object, go to **Directory > Users**, and click the edit icon beside the name.
+To modify a user object, go to **Directory > Users**, and click the edit icon beside the name. You can also go into [user details](#view-user-details), and click **Edit**.
+
+### Assign, modify, or remove permissions for a user
 
-You can also go into [user details](#view-user-details), and click **Edit**.
+You can grant a user specific global or object-level permissions. Alternatively, you can add a user to a group that has the appropriate permissions, and the user inherits all of the group's permissions.
```

Comment:
Suggestion: Provide links to the documentation for these concepts?

Reply:
Yes, now that we have those topics! I forgot to go back and add them, thanks for reminder.